wpk/run.sh

```diff
@@ -175,7 +175,9 @@ main() {
 
         # Compile agent
         make -C src -j ${JOBS} TARGET=${BUILD_TARGET} || exit 1
-        # Clean unuseful files
+        rm -rf ./src/symbols/.gitignore
+        tar -C src/symbols -czf ${OUTDIR}/wazuh-${BUILD_TARGET}-linux-x86_64-debug-info-${WAZUH_VERSION}-${REVISION}.tar.gz .
```

Comment:
Current output is `wazuh-agent-linux-x86_64-debug-info-v4.4.0-1.tar.gz`. Version should not contain `v` prefix

Reply:
Nevermind.`v` was ok according to wpk name convention